### PR TITLE
Initial styling to the navigation bar

### LINKED ISSUE #28: Navigation styling
**Is your feature request related to a problem? Please describe.**
Styling the navigation

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Added initial styling to the navigation bar

## Purpose
Initial styling to enhance user experience

Closes #28

### CODE REVIEW

File: pokeapp/src/styling/components/_navigation.scss [hunk 1]
```diff
@@ -4,5 +4,58 @@
...
+        border-radius: 8.2px;
+        padding: 0 5px;
+    }
+}
+
+
+
+
+
+
```
Comment (BlackCubes) [inline pokeapp/src/styling/components/_navigation.scss:61]: Using sass. Noice!

**Review by BlackCubes — APPROVED** [review]
My MAN! Looks gewd!

File: pokeapp/src/components/Navigation.js [hunk 2]
```diff
@@ -3,9 +3,26 @@ import React from "react";
...
+                    <div className="card-links">
+                        <h4>Pokedex</h4>
+                        <h4>MyPoke</h4>
+                    </div> 
+                    <div className="coin-link">
+                        <div className="coin-link-total">
+                            <h4>$1,357</h4>
+                        </div>
+                        <div className="coin-link-earn">
+                            <h4>Earn Coins</h4>
```
Comment (Lap343) [inline pokeapp/src/components/Navigation.js:21]: Earn coins??? Is this a game??? Excited to see

**Review by Lap343 — APPROVED** [review]
Good code. Can't wait to see ot

File: pokeapp/src/components/Navigation.js (hunk 2 above)
Comment (JorgeLVilla) [inline pokeapp/src/components/Navigation.js:21]: Excited to see it live!

**Review by JorgeLVilla — APPROVED** [review]
Great job! Looks good to me.

File: pokeapp/src/styling/components/_navigation.scss (hunk 1 above)
Comment (JayDampitan) [reply pokeapp/src/styling/components/_navigation.scss:61]: My Guy Eli, thanks bromigo!

**Pull request merged**